packages/board/src/svelte/BoxRender.svelte

```diff
@@ -1,18 +1,21 @@
 <script lang="ts">
-    import { BOX_THICKNESS, BOX_THICKNESS_HALF } from "@sudoku-studio/board-utils";
+    import { arrayObj2array, getBorderPath, idxMapToKeysArray, BOX_THICKNESS, BOX_THICKNESS_HALF } from "@sudoku-studio/board-utils";
     import type { StateRef } from "@sudoku-studio/state-manager";
 
     export let id: string;
-    export let ref: StateRef;
+    export let ref: StateRef;    // schema.BoxElement
```

Comment:
Would it make sense to rename this to `RegionRender`? And maybe also renaming the constants being imported from `board-utils` to `REGION_BORDER_THICKNESS` and `REGION_BORDER_THICKNESS_HALF` as well?

Reply:
Sure. should I rename `BoxInfo` and `BoxElement` as well?

Reply:
Yeah, I think that'd be great

Reply:
My plan was to rename `BoxElement` to `RegionElement`, but that already exists for max, min, even, and odd. Any thoughts on a better name?

Reply:
Maybe `GridRegionElement` or `SudokuRegionElement`?